app/controllers/application.js

```diff
@@ -207,15 +207,15 @@ export default class ApplicationController extends Controller.extend(
   }
 
   @tracked
-  openModal = !window.localStorage.hideMessage;
+  openModal = !window.localStorage.hidePopupMessage;
```

Comment:
I do agree that "hidePopupMessage" is slightly more specific. I'm flagging that anyone that had previously dismissed the message when it was called "hideMessage" will need to re-dimiss it under the "hidePopupMessage" name. Maybe this is okay because we've also updated the banner with more content?

Reply:
Yes, that was the intended behavior in renaming the local storage key, to cause the pp up to show up again with the new features / messaging.